packages/blocks/src/__internal__/rich-text/rich-text.ts

```diff
@@ -91,7 +91,8 @@ export class RichText extends LitElement {
     });
 
     page.attachRichText(model.id, this.quill);
-    page.awareness.updateLocalCursor();
+    // TODO it can cause cursor flickering, by need to evaluate impact
+    // page.awareness.updateLocalCursor();
```

Comment:
After removing the `asyncFocusRichText` after the transform block, the `updateLocalCursor` will cause cursor flickering after transform.

Is the update here necessary? @doodlewind